pkgs/applications/misc/kupfer/default.nix

```diff
@@ -30,6 +30,8 @@ buildPythonApplication rec {
   buildInputs = [ hicolor-icon-theme docutils libwnck3 keybinder3 ];
   propagatedBuildInputs = [ pygobject3 gtk3 pyxdg dbus-python pycairo ];
 
+  strictDeps = false;
```

Comment:
Could you also add a comment with explanation, pointing to https://github.com/NixOS/nixpkgs/issues/56943

Reply:
I hope the comment I have added is sufficient.